lib/variable/include/scipp/variable/bin_array_model.h

```diff
@@ -118,6 +120,13 @@ bool BinArrayModel<T>::equals(const Variable &a, const Variable &b) const {
          equals_impl(a.values<bucket<T>>(), b.values<bucket<T>>());
 }
 
+template <class T>
+bool BinArrayModel<T>::equals_nan(const Variable &a, const Variable &b) const {
+  // TODO This implementation is slow since it creates a view for every bucket.
```

Comment:
How slow? Open an issue?

Reply:
This is not a new comment (see a couple lines above). It has not showed up in practice (maybe because we don't have bins in coords, and don't compare things much other wise outside unit tests?).

Reply:
I just wanted to make sure this isn't lost in the source code but properly traced if it were important.